src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int SUPPLY_BUY_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SPLIT_REGEX = ",";
+    private StringBuilder builder = new StringBuilder();
```

Comment:
I didn't mean that you should move builder to class
I meant - how do you use it? do you need it at all?